funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         endpoint_dir=".",
         funcx_client: FuncXClient | None = None,
         result_store: ResultStore | None = None,
+        max_fail_count: int = 5,
```

Comment:
This variable naming isn't super clear, maybe something like `max_reconnect_count`? . It doesn't look like this config option isn't exposed to the user. Is that intentional?

Reply:
Not the first to observe this.  I'll address.  Thanks!